priv/repo/seeds.exs

```diff
@@ -13,15 +13,15 @@
 alias CbusElixir.Repo
 alias CbusElixir.App.Meeting
 
-start = Timex.beginning_of_month(Timex.now)
+start_date = Timex.beginning_of_month(~N[2018-03-06T00:00:00])
```

Comment:
Not sure that seeds is really needed at this time other than if we had to redo everything on a new server for some reason. This was to set up the initial 100 months of meetings in the system.

Reply:
What if you are brand new to the cbus_elixir proj and have never seeded the db before. I believe the query test would break since seed data would only include meetings from today on and not retro ones.